src/core/tools/generateImageTool.ts

```diff
@@ -12,7 +12,12 @@ import { OpenRouterHandler } from "../../api/providers/openrouter"
 import { KilocodeOpenrouterHandler } from "../../api/providers/kilocode-openrouter"
 
 // Hardcoded list of image generation models for now
-const IMAGE_GENERATION_MODELS = ["google/gemini-2.5-flash-image", "openai/gpt-5-image", "openai/gpt-5-image-mini"]
+const IMAGE_GENERATION_MODELS = [
+	"google/gemini-2.5-flash-image",
+	"google/gemini-3-pro-image-preview", // kilocode_change
```

Comment:
I would extract a KILOCODE_IMAGE_GENERATION_MODELS to make the merge easier, and splat that at the start of this array